portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,115 @@
+                </div>
+                <div class="mySlides fade" style="display: none;">
+                    <div class="numbertext">3 / 8</div>
+                    <img src="/images/20190329_232240.jpg" style="width:100%">
+                    <div class="text">Custom PDB inside payload</div>
+                </div>
+                <div class="mySlides fade" style="display: none;">
+                    <div class="numbertext">4 / 8</div>
+                    <img src="/images/20190330_100859.jpg" style="width:100%">
+                    <div class="text">Launch Day with other schools</div>
+                </div>
+                <div class ="mySlides fade" style="display: none;">
+                    <div class="numbertext">5 / 8</div>
+                    <img src="/images/IMG_0268.jpg" style="width:100%">
+                    <div class="text">Quick selfie in Armstrong FRC Hanger</div>
+                </div>
+                <div class="mySlides fade" style="display: none;">
+                    <div class="numbertext">6 / 8</div>
+                    <img src="/images/IMG_0294.jpg" style="width:100%">
+                    <div class="text">Rover team building project with Lego Mindstorms</div> 
+                </div>
+                <div class="mySlides fade" style="display: none;">
+                    <div class="numbertext">7 / 8</div>
+                    <img src="/images/33bf5a62-61be-4976-905d-5f8cf13c8172.jpg" style="width:100%">
+                    <div class="text">My awesome Team! We won 1st place</div>
+                </div>
+                <div class="mySlides fade" style="display: none;">
+                    <div class="numbertext">8 / 8</div>
+                    <img src = "/images/IMG_2719.jpg" width="800" height="600">
+                    <div class="text">A temple I visited in Fukuoka, Japan</div>
+                </div>
+                <!-- Next and previous buttons -->
+                <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+                <a class="next" onclick="plusSlides(1)">&#10095;</a>
+            </div><br>
+            <!-- The dots/circles -->
+            <div style="text-align:center">
+                <span class="dot" onclick="currentSlide(1)"></span>
+                <span class="dot" onclick="currentSlide(2)"></span>
+                <span class="dot" onclick="currentSlide(3)"></span>
+                <span class="dot" onclick="currentSlide(4)"></span>
+                <span class="dot" onclick="currentSlide(5)"></span>
+                <span class="dot" onclick="currentSlide(6)"></span>
+                <span class="dot" onclick="currentSlide(7)"></span>
+                <span class="dot" onclick="currentSlide(8)"></span>
+            </div>
+      </section>
+      <div>
+          <h4>Leave a Comment Below</h4>
+          <p>What did you think of my page</p>
+          <div onload="getDataFromJson()"></div>
```

Comment:
Will this onload work? My understanding is that onload does not work on divs? I could be wrong